modules/javafx.graphics/src/main/java/com/sun/javafx/font/PrismFontFile.java

```diff
@@ -607,7 +607,9 @@ private void init(String name, int fIndex) throws Exception {
              * any file read implementation which doesn't have random access.
              */
             initNames();
-
+            if (name != null) {
+                fullName = name;
+            }
```

Comment:
So I suppose this is predicated on the assumption that the name being passed in is retrieved from the platform for *all* platforms (not just Mac) so we are sure it is a valid name ?
I'd be interested to see what the difference is in the full names we then display on each platform and whether locale matters .. the name got from the platform might be a locale name and that may not have been the intention here ..  and is it possible "name" has already had toLowerCase() called on it ?